packages/fx-core/resource/package.nls.json

```diff
@@ -913,7 +914,8 @@
   "driver.oauth.error.OauthIdentityProviderInvalid": "Invalid identity provider 'MicrosoftEntra'. Ensure the OAuth authorization endpoint in the OpenAPI specification file is for Microsoft Entra.",
   "driver.oauth.log.successCreateOauth": "OAuth registration created successfully with id %s!",
   "driver.oauth.error.domainInvalid": "Maximum %d domains allowed per OAuth registration.",
-  "driver.apiKey.error.oauthAuthInfoInvalid": "Unable to parse OAuth2 authScheme from spec. Make sure your API specification is valid.",
+  "driver.oauth.error.oauthAuthInfoInvalid": "Unable to parse OAuth2 authScheme from spec. Make sure your API specification is valid.",
+  "driver.oauth.error.oauthAuthMissingInSpec": "No API in the OpenAPI specification file contains OAuth authentication with the name %s.",
```

Comment:
Proposed content: No API in the OpenAPI spec uses OAuth authentication named '%s'. Please verify the name in the specification.

Reply:
Thanks, updated